mailbox/backup/src/test/java/org/apache/james/mailbox/backup/DefaultMailboxBackupTest.java

```diff
@@ -305,6 +306,70 @@ void backupAccountWithTwoMailboxWithAnnotationsThenRestoringItInUser2AccountShou
         backupAccountWithTwoMailboxWithAnnotationsThenRestoringItInUser2AccountShouldRestoreAllElements();
     }
 
+    private void backupAccountWithOneMailboxAndTwoMessageThenRestoringItInUser2AccountShouldCreateOneMailboxWithTwoMessage() throws Exception {
+        createMailBoxWithMessages(sessionUser, MAILBOX_PATH_USER1_MAILBOX1, getMessage1AppendCommand(), getMessage2AppendCommand());
+
+        ByteArrayOutputStream destination = new ByteArrayOutputStream(BUFFER_SIZE);
+        backup.backupAccount(USER1, destination);
+
+        InputStream source = new ByteArrayInputStream(destination.toByteArray());
+        Mono.from(backup.restore(USER2, source)).block();
+
+        List<DefaultMailboxBackup.MailAccountContent> content = backup.getAccountContentForUser(sessionOtherUser);
+
+        assertThat(content).hasSize(1);
+        DefaultMailboxBackup.MailAccountContent mailAccountContent = content.get(0);
+        Mailbox mailbox = mailAccountContent.getMailboxWithAnnotations().mailbox;
+        assertThat(mailbox.getName()).isEqualTo(MAILBOX_1_NAME);
+        assertThat(mailAccountContent.getMessages().count()).isEqualTo(2);
+    }
+
+    @Disabled
+    @Test
+    void backupAccountWithOneMailboxAndTwoMessageThenRestoringItInUser2AccountShouldCreateOneMailboxWithTwoMessageInMemory() throws Exception {
+        backupAccountWithOneMailboxAndTwoMessageThenRestoringItInUser2AccountShouldCreateOneMailboxWithTwoMessage();
```

Comment:
- The test name is too long, it's hard to read
 - Two similar tests can be simplified into one by using JUNIT 5 parameterized test

IDK for other people, but this is accepted for me
```java
@ParameterizedTest
@ValueSource(ints = {MEMORY_STORE, FILE_STORE})
 void restoreToDifferentUserShouldCreateMailboxesAndMessages(int loaderThreshold) {
        applyLoaderThreshold(loaderThreshold);
        .... assertions
 }
```